Sources/HttpPipeline/SharedMiddlewareTransformers.swift

```diff
@@ -164,9 +164,9 @@ private func makeHttps(url: URL) -> URL? {
 /// Transforms middleware into one that logs the request info that comes through and logs the amount of
 /// time the request took.
 public func requestLogger(
-  _ middleware: @escaping Middleware<StatusLineOpen, ResponseEnded, Prelude.Unit, Data>
+  _ middleware: @escaping Middleware<StatusLineOpen, ResponseEnded, Never, Never, Prelude.Unit, Data>
```

Comment:
This is our only middleware with a concrete type. Kind of neat, though, to see that a logging middleware probably only makes sense if it captures _all_ responses.